src/modules/git_branch.rs

```diff
@@ -37,6 +37,15 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
     let branch_name = repo.branch.as_ref()?;
     let mut graphemes: Vec<&str> = branch_name.graphemes(true).collect();
 
+    for ignore_branch in config.ignore_branches {
```

Comment:
Not a blocker, but maybe it would be better to use an iterator and the `branch_name` here, something like

```rust
let is_ignored_branch = config.ignored_branches.iter()
    .any(|ignore_branch| branch_name.eq(ignore_branch));

if is_ignored_branch {
    return None;
} 
```

(I am on my phone so the above probably has something wrong with it but hopefully you get the idea).